src/main/java/uk/gov/hmcts/reform/civil/service/referencedata/LocationRefDataService.java

```diff
@@ -45,6 +45,20 @@ public List<String> getCourtLocations(String authToken) {
         return new ArrayList<>();
     }
 
+    public List<LocationRefData> getCourtLocationsAsLocationRefData(String authToken) {
```

Comment:
don't think you need this anymore since it's in master?

Reply:
I don't know why it still appears as new code since it is your code